UndertaleModTool/Scripts/Builtin Scripts/ShowRoomName.csx

```diff
@@ -1,30 +1,50 @@
+
 // Adds room ID and name display under the debug mode timer display
 
 EnsureDataLoaded();
-
+ScriptMessage("Show room name and ID in debug mode\nby krzys_h, Kneesnap\n\n\nDeltarune Ch2 Patch By GattoDev");
```

Comment:
I think that the credits in this message are redundant

Reply:
Oh, nevermind, I see now that the message was there, and you just added yourself.

Reply:
that was stupid and remoce that i believe so